submissions/lillianNwaoha/html/index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>Lily's Place</title>
+    </head>
+    <body>
+        <h1> LILY'S MAKEUP STORE </h1>
+        <a href="home">Home</a>
+        <a href="product">Products</a>
+        <a href="contact">Contact Us</a> 
+
+        <h2>HOME TO THE BEST MAKEUP PRODUCTS</h2>
+        <h3>FOUNDATION PRODUCTS</h3>
+        <img src="./images/img1.jfif" alt="born this way">
+        <h4>PRO MATTE FOUNDATION <br>N6,000</h4>
+        
+        <p>DESCRIPTION: With a soft and suede-like finish, Born this way foundation will have you covered and shine free all day. Creamy,liquid formula provides smooth and buildable coverage for both day and night time wear.</p>
```

Comment:
This text is too long to be on one line. Kindly use your `enter` key on your keyboard to break it down into a new line